lib/page/home/widget/home_surveys_indicators_widget.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:page_view_dot_indicator/page_view_dot_indicator.dart';
+import 'package:survey/resource/dimens.dart';
+
+class HomeSurveysIndicatorsWidget extends StatelessWidget {
+  final int surveysAmount;
```

Comment:
`surveysAmount` is weird naming, what do you think about `surveysCount` or `surveysLength`? 🤔